packages/v4/src/content/design-guidelines/usage-and-behavior/pagination/pagination.md

```diff
@@ -4,42 +4,48 @@ section: usage and behavior
+Use full pagination unless you're restricted to a narrow workspace. Full pagination occupies more toolbar territory, so you'll have limited room for other items like bulk selectors, buttons, filters, or input fields. 
+
+[Overflow menus](http://patternfly.org/v4/design-guidelines/usage-and-behavior/toolbar#overflow-menus) allow you to incorporate these toolbar actions using less horizontal space. Toolbar actions automatically form into an overflow menu at specific breaking points. Learn how to customize these breakpoints in our [Toolbar](/design-guidelines/usage-and-behavior/toolbar) documentation.
+
+Desktop and mobile views connect with the same breakpoint specifications. On mobile, full pagination condenses to an item count. As a toolbar item, you can add and keep the item count in two ways: in both desktop and mobile views, or just in mobile view.
 
 Desktop full pagination in toolbar
 ![full-pagination](./img/full-pagination.png)
 
-**1. Items in view:** Selects the item count per page (how many total items are listed per page).
+**1. Items in view:** Allows the user to select the item count (number of listed items) per page.
 
-**2. Full pager:** Gives the user all pagination options necessary including page-back and page-forward, page-first and page-last, and a manual option to type in the page number.
+**2. Full pager:** Supplies all necessary pagination options including page-back and page-forward, page-first and page-last, and a manual option to type in a desired page number.
 
 Mobile full pagination in toolbar
 ![mobile-pagination](./img/mobile-pagination.png)
 
-**1. Items in view:** Pagination controls are hidden in the top toolbar at this screen size. Only the item count is shown to indicate items that are currently in view.
+**1. Items in view:** At this screen size, the top toolbar only shows the item count to indicate items currently in view. It hides all other pagination controls. 
 
-**2. Full pager:** In the bottom toolbar, this gives the user all pagination options necessary including page-back and page-forward, page-first and page-last, and a manual option to type in the page number.
+**2. Full pager:** Supplies the same pagination options as the desktop full pager, but in the bottom toolbar.
 
 ## Compact pagination
 
-We recommend using the compact pagination variation when you do have spatial limitations within the top toolbar. This variation allows for many more actions to be placed within the toolbar. The compact variation also condenses into only an item count upon mobile.
+Compact pagination fits more actions into top toolbars with limited space. Choose this compact variation when your top toolbar contains many items. As with full pagination, the compact variation condenses to an item count for mobile views.
```

Comment:
I would maybe follow "Choose this compact variation when your top toolbar contains many items" with "or looks very crowded", or something along those lines to emphasize that this helps in situations where you don't want the toolbar to look super overloaded!

Reply:
Could we say "or looks overcrowded"?

Reply:
yep, that works!